src/NewChain/AuxiliaryChainInteract.ts

```diff
@@ -482,6 +485,7 @@ export default class AuxiliaryChainInteract {
     originOstGatewayAddress: string,
     originHeight: string,
     originStateRoot: string,
+    originChainId?: string,
```

Comment:
Why this needs to be added if in constructor there is already **originChainId** ?

Reply:
Because it's numeric ID, that value could be both chainID as well as chain Identifier(like 'ropsten')